scripts/copy_output.lua

```diff
@@ -66,6 +67,20 @@ for _, images in pairs(lib.images) do
 	end
 end
 
+-- copy kernel image with debug symbols
+local kernel_debug_glob = string.format('%s/\0-%s-kernel-debug.tar.zst',
+        env.GLUON_DEBUGDIR,
+        target)
+lib.exec {'rm', '-f', kernel_debug_glob}
+local kernel_debug_source = string.format('openwrt/bin/targets/%s/kernel-debug.tar.zst',
+        bindir)
+local kernel_debug_dest = string.format('%s/%s-%s-%s-kernel-debug.tar.zst',
```

Comment:
The glob and dest patterns should have `gluon-` before the site code.

Reply:
Done.
with:
```
$ make all GLUON_TARGET=ar71xx-generic V=s -j10 GLUON_RELEASE=v2020.1-182-g93cdd052-20200603xxxx-parker_beta
```

I now get an output of 
```
$ ls output/debug
gluon-ffbs-next-v2020.1-182-g93cdd052-20200603xxxx-parker_beta-ar71xx-generic-kernel-debug.tar.zst
```

An of course it is cleand up before the next build of this target.